Tue Feb 27 14:55:25 2024
Windows-10-10.0.22621
Ren'Py 8.1.3.23091805
 

Bootstrap to the start of init.init. took 0.05s
Early init. took 0.00s
Loader init. took 0.12s
Loading error handling. took 0.05s
Loading script. took 0.60s
Loading save slot metadata. took 0.07s
Loading persistent. took 0.00s
intit main()
Running init code. took 0.11s
Loading analysis data. took 0.02s
Analyze and compile ATL. took 0.00s
Reloading save slot metadata. took 0.04s
Index archives took 0.00s
Backing up script files to 'C:\\Users\\carol\\AppData\\Roaming/RenPy/backups\\Production':
Dump and make backups. took 0.09s
Cleaning cache. took 0.00s
Making clean stores. took 0.00s
Initial gc. took 0.04s
DPI scale factor: 1.250000
nvdrs: Loaded, about to disable thread optimizations.
nvdrs: Disabled thread optimizations.
Creating interface object. took 0.39s
Cleaning stores took 0.00s
Init translation took 0.04s
Build styles took 0.00s
Load screen analysis took 0.01s
Analyze screens took 0.02s
Save screen analysis took 0.04s
Prepare screens took 0.12s
Save pyanalysis. took 0.00s
Save bytecode. took 0.04s
Running _start took 0.00s
Performance test:
Interface start. took 0.11s

Initializing gl2 renderer:
primary display bounds: (0, 0, 1920, 1080)
swap interval: 1 frames
Windowed mode.
Vendor: "b'NVIDIA Corporation'"
Renderer: b'NVIDIA GeForce RTX 3060 Laptop GPU/PCIe/SSE2'
Version: b'4.6.0 NVIDIA 546.01'
Display Info: None
Screen sizes: virtual=(3840, 2160) physical=(1920, 991) drawable=(1920, 991)
Maximum texture size: 4096x4096
Enabled Forum
Stoped Forum

Full traceback:
  File "game/story/introduction/hw_day_2.rpy", line 211, in script call
    call d2_dm_wicker
  File "game/story/introduction/hw_day_2.rpy", line 469, in script
    legendforce12_nvl "THERE ARE A TON OF SKILLED PLAYERS BESIDES MYSELF HERE, SO IT'S NOT GOING TO BE EASY TO PLACE WELL. I HAVEN'T SEEN YOU ON THE LEADERBOARDS BEFORE AND ANY% IS HARD, SO YOU BETTER PRACTICE."
  File "C:\Users\carol\Downloads\renpy-8.1.3-sdk\renpy\ast.py", line 930, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "C:\Users\carol\Downloads\renpy-8.1.3-sdk\renpy\exports.py", line 1463, in say
    what = what % tag_quoting_dict
ValueError: unsupported format character 'I' (0x49) at index 155

While running game code:
  File "game/story/introduction/hw_day_2.rpy", line 211, in script call
    call d2_dm_wicker
  File "game/story/introduction/hw_day_2.rpy", line 469, in script
    legendforce12_nvl "THERE ARE A TON OF SKILLED PLAYERS BESIDES MYSELF HERE, SO IT'S NOT GOING TO BE EASY TO PLACE WELL. I HAVEN'T SEEN YOU ON THE LEADERBOARDS BEFORE AND ANY% IS HARD, SO YOU BETTER PRACTICE."
ValueError: unsupported format character 'I' (0x49) at index 155
Resetting cache.
Bootstrap to the start of init.init. took 98.52s
Early init. took 0.00s
Loader init. took 0.11s
Loading error handling. took 0.05s
Loading script. took 0.36s
Loading save slot metadata. took 0.09s
Loading persistent. took 0.00s
intit main()
Running init code. took 0.05s
Loading analysis data. took 0.03s
Analyze and compile ATL. took 0.00s
Reloading save slot metadata. took 0.08s
Index archives took 0.00s
Backing up script files to 'C:\\Users\\carol\\AppData\\Roaming/RenPy/backups\\Production':
Dump and make backups. took 0.10s
Cleaning cache. took 0.00s
Making clean stores. took 0.00s
Initial gc. took 0.07s
DPI scale factor: 1.250000
nvdrs: Loaded, about to disable thread optimizations.
nvdrs: Disabled thread optimizations.
Creating interface object. took 0.39s
Cleaning stores took 0.00s
Init translation took 0.06s
Build styles took 0.00s
Load screen analysis took 0.01s
Analyze screens took 0.02s
Save screen analysis took 0.03s
Prepare screens took 0.11s
Save pyanalysis. took 0.00s
Save bytecode. took 0.03s
Running _start took 0.00s
Performance test:
Interface start. took 0.18s

Initializing gl2 renderer:
primary display bounds: (0, 0, 1920, 1080)
swap interval: 1 frames
Windowed mode.
Vendor: "b'NVIDIA Corporation'"
Renderer: b'NVIDIA GeForce RTX 3060 Laptop GPU/PCIe/SSE2'
Version: b'4.6.0 NVIDIA 546.01'
Display Info: None
Screen sizes: virtual=(3840, 2160) physical=(1920, 991) drawable=(1920, 991)
Maximum texture size: 4096x4096

Full traceback:
  File "game/story/introduction/hw_day_2.rpy", line 211, in script call
    call d2_dm_wicker
  File "game/story/introduction/hw_day_2.rpy", line 507, in script
    python:
  File "C:\Users\carol\Downloads\renpy-8.1.3-sdk\renpy\ast.py", line 1138, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\carol\Downloads\renpy-8.1.3-sdk\renpy\python.py", line 1122, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/story/introduction/hw_day_2.rpy", line 508, in <module>
    if d2_t0read != True:
NameError: name 'd2_t0read' is not defined

While running game code:
  File "game/story/introduction/hw_day_2.rpy", line 211, in script call
    call d2_dm_wicker
  File "game/story/introduction/hw_day_2.rpy", line 507, in script
    python:
  File "game/story/introduction/hw_day_2.rpy", line 508, in <module>
    if d2_t0read != True:
NameError: name 'd2_t0read' is not defined
